File: be/SpringBoot/src/main/java/com/common/dto/response/TenantDTO.java
package com.common.dto.response;

import com.common.model.enumerate.EntityStatus;
import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

import java.time.LocalDateTime;

/**
 * DTO đại diện cho dữ liệu Tenant.
 *
 * @created_at 2026-07-31
 * @author txhoan
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class TenantDTO {

    Long id;

    String name;

    String code;

    String domain;

    EntityStatus status;

    LocalDateTime createdAt;

    LocalDateTime modifiedAt;
}

File: be/.Net/Swe.BL/DTO/Response/TenantDTO.cs
using System;

namespace Swe.BL.DTO.Response;

/// <summary>
/// DTO đại diện cho dữ liệu Tenant.
/// </summary>
/// <created_at>2026-08-03</created_at>
/// <author>txhoan</author>
public class TenantDTO
{
    public long Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Code { get; set; } = string.Empty;
    public string? Domain { get; set; }
    public string Status { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public DateTime? ModifiedAt { get; set; }
}
